examples/star-wars/js/components/StarWarsApp.js

```diff
@@ -13,23 +13,72 @@
 import React from 'react';
 import Relay from 'react-relay';
 import StarWarsShip from './StarWarsShip';
+import AddShipMutation from '../mutation/AddShipMutation';
 
 class StarWarsApp extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      shipName: '',
+      factionId: 1
+    };
+  }
+
+  addship() {
```

Comment:
This is an event handler. Can we name it `handleAddShipClick` to make this clear? Alpha order it with the other handle-type methods.